src/frontend/src/common/redux/utils/groupUtils.ts

```diff
@@ -14,7 +14,15 @@ export const ensureValidGroup = async (): Promise<void> => {
 
   const { dispatch, getState } = store;
   const state = getState();
-  const rawUserInfo = await fetchUserInfo();
+  let rawUserInfo: RawUserRequest | null = null;
+  try {
+    rawUserInfo = await fetchUserInfo();
+  } catch (e) {
+    return;
```

Comment:
Nit: I generally prefer `return null` when it's being used to say "hey, don't render this component", it feels slightly clearer to me.